interface/patient_file/summary/stats_full.php

```diff
@@ -268,7 +268,7 @@ function newEncounter() {
                     }
 
                     echo " <tr class='" . attr($bgclass) . " detail' $colorstyle>\n";
-                    echo "  <td class='text-left " . attr($click_class) . "' id='" . attr($rowid) . "'>" . text($disptitle) . "</td>\n";
+                    echo "  <td class='text-left " . attr($click_class) . " text-decoration: underline' id='" . attr($rowid) . "'>" . text($disptitle) . "</td>\n";
```

Comment:
This is still incorrect. The declaration should be as follows:
`echo "  <td class='text-left " . attr($click_class) . "' style='text-decoration: underline' id='" . attr($rowid) . "'>" . text($disptitle) . "</td>\n";`

Reply:
Well it works so why is it incorrect?

Reply:
Seems like you are missing some single quote and style attribute.

Reply:
As I said it works - try it. But so does the `style` attribute - I changed it.